src/tests/template_srv.ts

```diff
@@ -0,0 +1,105 @@
+import * as _ from 'lodash';
+import { ScopedVars, TimeRange, VariableModel } from '@grafana/data';
+
+export class TemplateSrv {
```

Comment:
I don't think you need all this class definition and code. Can't you just mock the template srv as done in src/QueryEditor.test.tsx?

Reply:
yeah, this sounds too much code to maintain here for the tests